### ISSUE TITLE
Wildcards in file options

### ISSUE THREAD

**Issue by zomars** [dciccale/grunt-processhtml#71]
I'm trying to use wildcards in the options arguments in the grunt file, but it seems it doesn't process properly.
```
files: {
        'dest/index.html': ['index.html']
}
```
I want to be able to do this:
```
files: {
        'dest/*.html': ['*.html']
}
```

**Comment by dciccale** [comment]
Please read carefully grunt documentation, plus see htis other issues solved: #45 #33 #29

**Comment by zomars** [comment]
Sorry about that. And thanks again.
